app/(gcforms)/[locale]/(form administration)/form-builder/components/shared/conditionals/AddOther.tsx

```diff
@@ -71,7 +71,11 @@ export const AddOther = ({
 
   return (
     <>
-      <Button className="!m-0 !mt-4 inline" theme={"secondary"} onClick={addOther}>
+      <Button
```

Comment:
Pushed a commit here for you

https://github.com/cds-snc/platform-forms-client/pull/4002/commits/fee466d0886a086bb1a30cf5908afae8fea67f9c